eth/calltracer/calltracer.go

```diff
@@ -4,10 +4,10 @@ import (
 	"encoding/binary"
 	"sort"
 
-	"github.com/holiman/uint256"
 	libcommon "github.com/gateway-fm/cdk-erigon-lib/common"
 	"github.com/gateway-fm/cdk-erigon-lib/common/length"
 	"github.com/gateway-fm/cdk-erigon-lib/kv"
+	"github.com/holiman/uint256"
```

Comment:
This may cause upstream merge pain, worth reverting :).

Reply:
Fixed https://github.com/0xPolygonHermez/cdk-erigon/pull/948/commits/15ca9617538e54bae6edfd4b0684fe289626f504